catalog/monitors/ec2-cpu-utilization.yaml

```diff
@@ -0,0 +1,36 @@
+# The official Datadog API documentation with available query parameters & alert types:
```

Comment:
prob move this EC2-related monitors to https://github.com/cloudposse/terraform-datadog-monitor/blob/master/catalog/monitors/ec2.yaml